testgrid/config.yaml

```diff
@@ -4102,6 +4106,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-scalability-stable2
   - name: gce-gpu-1.9
     test_group_name: ci-kubernetes-e2e-gce-gpu-stable2
+  - name: pod-preemption
+    test_group_name: ci-kubernetes-e2e-gce-pod-preemption
```

Comment:
eh, I don't think this is the right testgrid dashboard for your job :-)

Reply:
add to sig-scheduling dashboard I think?

Reply:
To be honest, I wasn't sure, the specific jobs like GPU added were part of sig-scheduling but they had their own dashboard group. So, I added them there. I won't minding moving them to sig-scheduling too..